tests/test_args.py

```diff
@@ -73,266 +77,282 @@ def log_capture(caplog, args):
     return LogCaptureObject(caplog.record_tuples, e)
 
 
-def test_version(capsys):
+def test_version(capsys: pytest.CaptureFixture[str]) -> None:
     c = capture(capsys, ["--version"])
     assert c.err == ""
     assert c.out.startswith(f"gcovr {__version__}")
-    assert c.exception.code == 0
+    assert c.exception is not None and c.exception.code == 0
 
```

Comment:
There is a "Codacy Static Code Analysis" warning about the use of `assert`, which is a false positive in the context of Pytest. (And questionable in other cases as well, since Python's "optimized" doesn't do what most people would think it does.)

Can this specific rule be disabled for all `tests/**.py` files?

Reply:
In pyproject.toml this is already excluded for the test modules but Codacy doesn't use the pyproject.toml for the tools. I wasn't able to request the support of pyproject.toml because of an error in there WEB interface.

Reply:
I'll deactivate this check on Codacy until pyproject.toml can be configured. We are running lint session in our CI and will get an error there.